templates/projects/view.tmpl

```diff
@@ -23,7 +23,7 @@
 				{{svg "octicon-trash"}}
 				{{ctx.Locale.Tr "repo.issues.label_delete"}}
 			</a>
-			<a class="item show-modal" data-modal="#new-project-column-item">
+			<a class="item show-modal" href="#" data-modal="#new-project-column-item">
```

Comment:
Wouldn't this introduce a unwanted URL hash clearing on click? I think menu items that are not links should be `<div class="item"/>` and Fomantic docs agree on it as well.

Reply:
Honestly, I don't know what's the right way here. That was suggested by @delvh, but I also might have misunderstood what was asked for.....

Reply:
It's quite simple, use `<a>` only when it is an actual link with a actual `href`. This is not such a case.

Reply:
Okay, I removed `href` completely....

Reply:
Fine with me for now, we have this kind of `<a>` abuse in many places. Ideally we should replace all that are not links with `<div>`, or if possible (I doubt it) `<button>`.

```
<a class="item show-modal" data-modal="#new-project-column-item"></a>
```

becomes
```
<div class="item show-modal" data-modal="#new-project-column-item"></div>
```

Not requesting this to be changed now but, let's leave this discussion open a bit for other to see.

Reply:
Ah now I understand what you mean..... does that work without additional changes having to be made? If so I'd also prefer it

Reply:
> Honestly, I don't know what's the right way here. That was suggested by @delvh, but I also might have misunderstood what was asked for.....

No need to do so. The framework code for `.show-modal` should be able to handle it automatically.

Reply:
> Ah now I understand what you mean..... does that work without additional changes having to be made? If so I'd also prefer it

Yes, as far as I can tell changing `<a>` to `<div>` should work without any additional changes.

Reply:
Changed it, seems to be unproblematic 👍